### PR TITLE
feat: add target_dir to action options

### PR DESCRIPTION
**Pull request by jdcmarques**
## Context
- Added `TARGET_DIR` option to allow to specify where we want the files to be copied to
  -  By default, it will only clean out the `TARGET_DIR`

requires #81 
implements #70

### CODE REVIEW

**Comment by marquesj2-ppb** [comment]
Hey @s0, can you take a look at this PR?

File: action/src/index.ts [hunk 1]
```diff
@@ -547,6 +551,11 @@ export const main = async ({
         .map((s) => s.trim())
         .filter((s) => s !== '');
       return globList;
+    } else if (env.TARGET_DIR) {
+      log.log(
+        `##[info] Removing all files from target dir ${env.TARGET_DIR} on target branch`
+      );
+      return [`${env.TARGET_DIR}/*`, '!.git'];
```
Comment (s0) [inline action/src/index.ts:558]: we probably want to make sure that we recursively delete everything from the target dir (and not just the immediate files)

File: README.md [hunk 2]
```diff
@@ -226,6 +254,13 @@ everything, and you will need to specify exactly what needs to be deleted.**
    !.git
    ```
 
+1. Default behaviour with a custom target directory:
+
+   ```
+   target_dir/*
```
Comment (s0) [inline README.md:260]:
```suggestion
   target_dir/**/*
```

**Review by s0 — COMMENTED** [review]
Awesome work, thanks so much! I've rebased it ontop of the other PRs I've now merged and your other work.

There may be one bug when deleting files, I'm going to create a unit test that checks this, and then address it, just to get this in :)

**Pull request merged**